### DISCUSSION TITLE
Dosage sensitivity scores

### DISCUSSION THREAD

**Discussion by lynnpais** [broadinstitute/seqr#2872] [General]
Regarding dosage sensitivity scores, I see it here:https://github.com/broadinstitute/seqr/blob/8b4419285dfac9365c5c500bbb87b89808c0cedd/reference_data/models.py#L143
but am not sure which version it is. Updated versions of the scores were released earlier this year: https://zenodo.org/record/6347673#.YuPXNOzML0o. Can we update to this version if we are not using it?

**Comment by hanars** [comment]
the current version in seqr if from January 2021. I have created an issue to track the work of updating to the latest version

**Comment by hanars** [comment]
Per https://github.com/broadinstitute/seqr/issues/2874 scores have been updated
